packages/flutter_android_lifecycle/pubspec.yaml

```diff
@@ -0,0 +1,60 @@
+name: flutter_android_lifecycle
+description: A new flutter plugin project.
+version: 0.0.1
+author:
```

Comment:
Update these fields and cleanup all the template comments.

Reply:
Done.